web-components/src/components/date-range-picker/DateRangePicker.ts

```diff
@@ -0,0 +1,72 @@
+import { customElementWithCheck } from "@/mixins/CustomElementCheck";
```

Comment:
please don't forget the disclaimer in all new files

Reply:
@yananym just to be 100% clear, same goes for all scss, tokens, test files?

Reply:
right